databricks/koalas/frame.py

```diff
@@ -1653,6 +1653,58 @@ def head(self, n=5):
 
         return DataFrame(self._sdf.limit(n), self._metadata.copy())
 
+    def pivot_table(self, values=None, index=None, columns=None,
+                    aggfunc='mean', fill_value=None):
+
+        if not isinstance(columns, str):
+            raise ValueError("columns should be string.")
```

Comment:
I know the code base is not 100% consistent in this regard, but error messages should probably start with uppercase letters.